### PR TITLE
Foodlist cont

### PR DESCRIPTION
**Pull request by Stanleeeeee**
In this commit I,

- created the food list using the Entity Relationship Diagram and wireframe provided

- created methods for foodlist

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -6,4 +6,7 @@ class User < ApplicationRecord
 
   has_many :recipes
   has_many :inventories
+  has_many :foods
+
+  validates :email, :password, presence: true
```
Comment (brahimdidi) [inline app/models/user.rb:11]: please remove this line as it might cause conflict

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -8,6 +8,21 @@
 #
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
+
```
Comment (brahimdidi) [inline config/initializers/devise.rb:11]: Optional:  basically did all these change please discard this file , so we do not have any conflict

File: config/routes.rb [hunk 3]
```diff
@@ -3,7 +3,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```
Comment (brahimdidi) [inline config/routes.rb:3]: please discard your changes here to avoid conflict after merging you can do it again

**Review by brahimdidi — APPROVED** [review]
Status Approved
however I would like you to make those optional change to avoid conflict
Thanks.

File: config/initializers/devise.rb (hunk 2 above)
Comment (Stanleeeeee) [reply config/initializers/devise.rb:11]: @brahimdidi Thanks for the review

**Pull request merged**